### PR TITLE
fix dop addr

### PR DESCRIPTION
**Pull request by QvodSoldier**
#### What type of this PR
/kind bug

#### What this PR does / why we need it:
fix dop registry to eventbox

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @sfwn

### CODE REVIEW

File: bundle/bundle.go [hunk 1]
```diff
@@ -293,3 +293,10 @@ func WithAllAvailableClients() Option {
 		b.urls.PutAllAvailable()
 	}
 }
+
+// WithCustom # TODO remove it after internal bundle invoke inside cp issue-manage adjusted
```
Comment (sfwn) [inline bundle/bundle.go:297]: remove TODO here.

move TODO to dop provider with `localhost:9527.`

File: modules/dop/initialize.go [hunk 2]
```diff
@@ -214,21 +213,21 @@ func (p *provider) initEndpoints(db *dao.DBClient) (*endpoints.Endpoints, error)
...
-		bundle.WithMonitor(),
-		bundle.WithCollector(),
-		bundle.WithHTTPClient(httpclient.New(
-			httpclient.WithTimeout(time.Second*15, time.Duration(conf.BundleTimeoutSecond())*time.Second), // bundle 默认 (time.Second, time.Second*3)
-		)),
-		bundle.WithKMS(),
-		bundle.WithCoreServices(),
-		bundle.WithDOP(),
-	)
+	// bdl.Init(
```
Comment (sfwn) [inline modules/dop/initialize.go:216]: remove useless code.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: bundle/bundle.go (hunk 1 above)
Comment (QvodSoldier) [reply bundle/bundle.go]: done

File: modules/dop/initialize.go (hunk 2 above)
Comment (QvodSoldier) [reply modules/dop/initialize.go]: done

**Review by sfwn — APPROVED** [review]

**Pull request merged**